PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/springcloud/deploy/SpringCloudDeploymentState.java

```diff
@@ -121,7 +121,7 @@ public AppResourceInner executeSteps(@NotNull RunProcessHandler processHandler
             throw new AzureExecutionException(String.format("Project '%s' cannot be found.",
                                                             springCloudDeployConfiguration.getProjectName()));
         }
-        String finalJarName = MavenUtils.getSpringBootFinalJarPath(project, targetProject);
+        String finalJarName = MavenUtils.getTargetFile(project, targetProject);
```

Comment:
Do we also need to remove below `validateSpringCloudAppArtifact(finalJarName)` validation?

Reply:
Done by remove some checks in validateSpringCloudAppArtifact